src/OctoshiftCLI.Tests/Octoshift/CliContextTests.cs

```diff
@@ -0,0 +1,29 @@
+using FluentAssertions;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.Octoshift;
+
+public class CliContextTests
+{
+    [Fact]
+    public void SetRootCommandShouldRemoveGhPrefix()
```

Comment:
Looks good, only suggestion is to use snake casing to be consistent with other tests
```suggestion
    public void Set_RootCommand_Should_Remove_Gh_Prefix()
```